src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,26 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+        {post.user && <UserInfo user={post.user} />}
+      </p>
+    </div>
+
+    <p className="PostInfo__body">{post.body}</p>
+
+    <hr />
+
+    {post.comments.length ? (
+      <CommentList comments={post.comments} />
+    ) : (
+      <b data-cy="NoCommentsMessage">No comments yet</b>
+    )}
+  </div>
```

Comment:
You can also move the expression about the existence of a comment to a separate constant. And then use this constant instead of this expression

Reply:
Sure, I can. So what? I dont want to. I want to use ternary operator here

Reply:
Here as well as in the past, mentors rejected me for this.